src/server/controllers/match.controller.js

```diff
@@ -33,14 +33,15 @@ export function requestMentoring(req, res, next) {
       })
       .then(mentor => {
         if (mentor) {
-          sendEmail(receiver, mailStrings.REQUEST_SUBJECT, mailStrings.REQUEST_HTML, matchData.content);
+          // TODO: Confirm method whether send mail or send in-app message.
+          // sendEmail(receiver, mailStrings.REQUEST_SUBJECT, mailStrings.REQUEST_HTML, matchData.message);
```

Comment:
If it's not confirmed yet, I think it should be as it was.
Don't think it needs to a comment.